test-clients/src/main/java/com/hedera/services/bdd/suites/autorenew/AccountAutoRenewalSuite.java

```diff
@@ -123,7 +124,8 @@ private HapiApiSpec accountAutoRenewal() {
 						getAccountInfo(autoRenewedAccount)
 								.has(accountWith()
 										.expiry(autoRenewedAccount, briefAutoRenew)
-										.balanceLessThan(initialBalance)),
+										.balanceLessThan(initialBalance))
+								.logged(),
```

Comment:
nit : Is logging needed ?

Reply:
Since we are in the early phases of arutoRenew testing and validation, I would like to keep logging on for manual validation purposes when JRS validators fails. We can turn this off and manually kick off a test with these on when we observer a failure and need more data